test/unit/codegen/codegen_llvm_instance_struct.cpp

```diff
@@ -131,51 +131,51 @@ SCENARIO("Instance Struct creation", "[visitor][llvm][instance_struct]") {
             size_t node_count_index = 19;
             // Check if the various instance struct fields are properly initialized
             REQUIRE(compare(instance_data.members[minf_index],
-                            generate_dummy_data<double>(minf_index, num_elements)));
+                            generate_dummy_data<float>(minf_index, num_elements)));
```

Comment:
I would maybe keep this test using `double` since this is the default use case and we don't have another test about this. Not sure why it was set to `use_single_precision` from the beginning. Probably some negligence from my side :innocent:

Reply:
Sure! I will change an execution test instead :)

Reply:
Thank you! :slightly_smiling_face: